app/models/testOnly/DocumentType.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.testOnly
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+sealed trait DocumentType {
+  val code: String
+  val description: Option[String]
+  val transportDocument: Option[Boolean]
+}
+
+object DocumentType {
+
+  implicit val reads: Reads[DocumentType] = (
+    (__ \ "code").read[String] and
+      (__ \ "description").readNullable[String] and
+      (__ \ "transportDocument").readNullable[Boolean]
```

Comment:
This logic was previously happening in the ETL service to derive this value, so do we need this here? Since we're calling the code lists separately from the frontend a 'document' just needs to be a code and a description